__tests__/async-await.test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+jest.mock('fs');
+
+const async = require('../lib/async-await.js');
```

Comment:
calling this `async` can become very confusing because `async` is a reserved keyword. I would caution against this kind of naming convention